pkg/api/defaulting_test.go

```diff
@@ -112,7 +112,7 @@ func TestDefaulting(t *testing.T) {
 		{Group: "rbac.authorization.k8s.io", Version: "v1alpha1", Kind: "RoleBindingList"}:        {},
 	}
 
-	f := fuzz.New().NilChance(.5).NumElements(1, 1).RandSource(rand.NewSource(1))
+	f := fuzz.New().NilChance(.5).NumElements(1, 1).RandSource(rand.NewSource(2))
```

Comment:
The defaulting tests don't pass unless I bump fuzzer's seed. It's this loop:

https://github.com/kubernetes/kubernetes/blob/master/pkg/api/defaulting_test.go#L148-L158

I won the lottery but it took me two hours to figure out what was going on so I don't feel very lucky.